pom.xml

```diff
@@ -62,8 +62,8 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <doxiaVersion>1.4</doxiaVersion>
-    <doxia-sitetoolsVersion>1.4</doxia-sitetoolsVersion>
+    <doxiaVersion>1.9.1</doxiaVersion>
+    <doxia-sitetoolsVersion>1.9.2</doxia-sitetoolsVersion>
```

Comment:
I did not try to run, but yes, it should be as siple as that: that's why the issue was marked as up-for-grabs